server/src/main/java/org/opensearch/extensions/ExtensionsManager.java

```diff
@@ -443,6 +449,54 @@ TransportResponse handleExtensionRequest(ExtensionRequest extensionRequest) thro
         }
     }
 
+    /**
+     * A separate transport action handler used to issue service accounts to extensions during initialization
+     * @param extension The extension to be issued a service account
+     */
+    public void issueServiceAccount(Extension extension) {
```

Comment:
Craig asked it to be  separate transport request:  https://github.com/opensearch-project/security/issues/3176.

Reply:
@peternied What if a token needs to be re-issued? I'm in favor of separate transport request

Reply:
Then the state should be updated, retransmitted, in a world where permissions of the extension change there should be a single kind of notification where the extension can know what it can/cannot do.  As I said we can always build it out later, but lets start small as this is an experimental feature.

Reply:
I agree that it should be more generic. `initializeExtensionSecuritySettings`?

In a draft PR I have open in the sdk there's a linked PR to my fork in core that shows a transport action called `initializeExtensionSecurity`.

https://github.com/cwperks/OpenSearch/pull/94/files#diff-c9a4accc046eb92a0e6a025970b3eacacc330f40bb55b02859df05a79710c8c8R420

Reply:
@peternied @cwperks what is the call?

Reply:
I will make it part of initialize like Peter and Owais suggested.